resources/dev.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -d "node_modules/.bin" ]; then
+  echo "Be sure to run \`npm install\` before developing GraphiQL."
+  exit 1
+elif [ ! -d "example/node_modules" ]; then
+  echo "Be sure to run \`npm install\` in the example directory before developing GraphiQL."
+  exit 1
+fi
+
+# Clean up dist in example
+rm -rf example/dist && mkdir -p example/dist
+
+# Boot the server
+node example/server.js &
+
+# Watch JS source files and recompile
```

Comment:
Main changes are here. This will watch the `src` file, output changes, babelify it, shim, export `GraphiQL`, and output to the standard `graphiql.js` file.